Makefile

```
@@ -1,10 +1,16 @@
-.PHONY: build doc
+.PHONY: build test doc lint bench
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)